handlers.go

```diff
@@ -184,6 +190,23 @@ func (s *server) logout(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/", http.StatusSeeOther)
 		return
 	}
+
+	// Check if the provider has a revocation_endpoint
+	revocationEndpoint, err := revocationEndpoint(s.provider)
```

Comment:
Go has an official guide called [effective go](https://golang.org/doc/effective_go.html#Getters) which instructs:
> Go doesn't provide automatic support for getters and setters. There's nothing wrong with providing getters and setters yourself, and it's often appropriate to do so, but it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner. The use of upper-case names for export provides the hook to discriminate the field from the method. A setter function, if needed, will likely be called SetOwner.

So I would expect that it's customary.

Reply:
Oh, I see, thanks for the reference in the guide.

Reading a bit more on this, my initial thought was that the function should probably be renamed to `RevocationEndpoint`. But, I see that capitalization is tied with whether a function is public or not.

Personally, I'd choose a different name for the variable, so that we don't shadow the function in this scope. But, if it's something that is common in Go, I'm okay with that.

Reply:
I think using a different name imposes a bigger cognitive overhead (at least for me) and since it's common to do `a = b.a()` it seems logical to also do `a = a()`.

Capitalization is indeed tied with function exporting and I'm a bit on the fence about this.
On the one hand, the best practice for now is for everything to be lowercased since everything is in the same package and we don't want anyone using this as a library.

On the other hand, if we someday decide to break this into separate packages, we'll have to change many function names. This may be good because it makes you think about the interface you want to expose, but I'm still a bit skeptical.

Reply:
Note that the difference between `a = b.a()` and `a = a()` is that you can't use `a()` in the same scope again, which is surprising and difficult to to debug (I should now, I've been bitten by this). Anw, we can leave it as is if you want to reuse the same name.

Reply:
You are right.
I changed to `_revocationEndpoint` as the best solution I could think of.